### DISCUSSION TITLE
基盤にmicroproを差し込むとPC側に認識されない

### DISCUSSION THREAD

**Discussion by poporirinu** [Yowkees/keyball#484] [Q&A]
お世話になっております
質問失礼致します。
Key ball39を作成しております。

基盤に差し込んだ状態でUSBを接続しても、マイコン本体のLEDが点灯せず、パソコンにも認識されません。基盤に差さない状態でしたら点灯いたします。
お忙しい所恐れ入りますがご教示願います。
![IMG_8270](https://github.com/Yowkees/keyball/assets/160295919/e524850a-f68b-4874-b1b1-6a31027d9748)
![IMG_8272](https://github.com/Yowkees/keyball/assets/160295919/54afd188-2e52-4fb1-af2d-e6a532551311)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

ProMicro単体では反応があるということですので、基板側に何か不具合があるかと思われます。
基板のはんだ付けも確認させていただきたいので、写真の添付をお願いしてもよろしいでしょうか。

**Comment by poporirinu** [comment]
ご回答ありがとうございます。
自身でハンダを温めなおしたり等色々試したのですが
症状は変わらずです。
ご確認お願い致します。
![IMG_8303](https://github.com/Yowkees/keyball/assets/160295919/7d48e39f-5002-4ca8-92c6-17660e4be67f)
![IMG_8302](https://github.com/Yowkees/keyball/assets/160295919/e87a878a-56f6-4faf-b558-62a909f4978f)
![IMG_8301](https://github.com/Yowkees/keyball/assets/160295919/9a3a43a8-f254-4037-955c-766bd51e4413)

ご確認お願い致します。

**Comment by Ryo624** [comment]
基板の写真を添付していただきましてありがとうございます。

光の加減等で気のせいかもしれませんが添付しました写真の赤い丸の端子のはんだが隣の端子に触れていないかご確認していただけますでしょうか。
また、赤い丸でも示しましたGNDと書かれた場所の端子のはんだが多いように見えますので、他の箇所のはんだのような形にしていただけますと幸いです。
![305758898-9a3a43a8-f254-4037-955c-766bd51e4413](https://github.com/Yowkees/keyball/assets/143698349/90287a4a-2a69-4d90-adc3-f54f6d21d99b)

もう一つ確認していただきたいのがProMicroの左右を入れ替えても同じ症状が起きるかのご確認をお願いできますでしょうか。

**Comment by poporirinu** [comment]
ご回答ありがとうございます。
はんだ量を減らしましたが症状は改善されませんでした。また隣同士の接触はございませんでした。
promicroを左右入れ替えても症状は改善されませんでした。
![IMG_8310](https://github.com/Yowkees/keyball/assets/160295919/5c9ec42b-d9bd-44f1-923e-b57992b1a133)
![IMG_8309](https://github.com/Yowkees/keyball/assets/160295919/a659379e-b71e-4988-8689-e56d6f3f53af)
![IMG_8308](https://github.com/Yowkees/keyball/assets/160295919/599281b6-047a-4ee9-a92f-dfacb14341f1)

**Comment by poporirinu** [comment]
因みに基盤と部品を右のみ別売りしていただく事は可能でしょうか？

**Comment by Ryo624** [comment]
白銀ラボのウェブショップから購入者様に向けて別売りを行っておりますので、購入をご希望の場合はウェブショップにありますお問い合わせフォームからご連絡いただければと思います。
